parlai/zoo/multimodal_blender/README.md

```diff
@@ -0,0 +1,145 @@
+# Model card: Multi-Modal BlenderBot
+
+Model card for the MMB DegenPos model described in [Multi-Modal Open-Domain Dialogue](TODO: add arXiv link).
+
+## Model details
+This model was trained to create a dialogue agent that can converse engangly about the content of an image as well as about general chitchat. It is a 2.7-billion-parameter Transformer sequence-to-sequence model based on the [BlenderBot](https://ai.facebook.com/blog/state-of-the-art-open-source-chatbot/) open-domain chatbot, trained with image features from [COCO Captions](https://cocodataset.org/) and [Image-Chat](https://parl.ai/projects/image_chat/), and using an image encoder from [Faster R-CNN](https://arxiv.org/abs/1506.01497?context=cs).
+
+## Training
+### Domain-adaptive pre-training
+To recreate a MMB DegenPos model, the base pre-trained BlenderBot3B model must first be further pre-trained on an image dataset, COCO Captions:
+```
+parlai tm \
+-t coco_caption \
+--include-rest-val True \
+--include-image-token False \
+--activation gelu \
+--attention-dropout 0.0 \
+--batchsize 128 \
+--dropout 0.1 \
+--fp16 True \
+--gradient-clip 0.1 \
+--label-truncate 128 \
+--log-every-n-secs 30 \
+--lr-scheduler reduceonplateau \
+--max-train-time 169344.0 \
+--model-parallel True \
+--model image_seq2seq \
+--init-model zoo:blender/reddit_3B/model \
+--dict-file zoo:blender/reddit_3B/model.dict \
+--embedding-size 2560 \
+--ffn-size 10240 \
+--n-decoder-layers 24 \
+--n-encoder-layers 2 \
+--n-heads 32 \
+--n-positions 128 \
+--variant prelayernorm \
+--text-truncate 128 \
+--truncate 128 \
+--dict-tokenizer bytelevelbpe \
+--fp16-impl mem_efficient \
+--optimizer adam \
+--update-freq 2 \
+--history-add-global-end-token end \
+--delimiter '  ' \
+--lr-scheduler-patience 3 \
+--warmup-updates 100 \
+--multitask-weights 1,1 \
+--relu-dropout 0.0 \
+--save-after-valid True \
+--skip-generation True \
+-lr 7e-06 \
+-vtim 1800 \
+-vmm min \
+-vmt ppl \
+-vp 10 \
+-vme 24000 \
+--image-fusion-type early \
+--n-segments 2 \
+--n-image-channels 100 \
+--model-file ${DOMAIN_PRETRAINED_MODEL_PATH}
+```
+
+### Fine-tuning
+After the round of domain-adaptive pre-training above, the model must be fine-tuned on the dialogue datasets and on the Image-Chat dataset, using a degendering teacher:
+```
+parlai tm \
+-t genderation_bias:controllable_task:blended_skill_talk,genderation_bias:controllable_task:convai2:normalized,genderation_bias:controllable_task:empathetic_dialogues,genderation_bias:controllable_task:wizard_of_wikipedia,genderation_bias:controllable_task:image_chat:Generation \
+--use-gendered-word-list True \
```

Comment:
remove this flag please (no longer in the teacher)

Reply:
cool, removing